plugins/inputs/consul/README.md

```diff
@@ -38,6 +38,17 @@ report those stats already using StatsD protocol if needed.
   # When tags are formatted like "key:value" with ":" as a delimiter then
   # they will be splitted and reported as proper key:value in Telegraf
   # tag_delimiter = ":"
+
+  ## Disable gathering tags from Consul on health checks
+  # This is very useful on large clusters with a lot of services and tags.
+  # This alleviates the situation of this telegraf running on multiple servers 
+  # for a holistic point of view, and there are thousands of checks with 10+ tags. 
+  # disable_tags = false
+
+  ## Disable gathering check id from Consul on health checks
+  # This is useful in dynamic environments, where check_id is generated,
+  # where most check_id's are some uuid-ish name with low meaning.
+  # disable_check_id = false
```

Comment:
Instead of this option, I suggest using `tagexclude = ["check_id"]`.  This is a general solution that can be used on any plugin.

Reply:
Sure, do you mind if I leave a note in the readme for this suggesting that for our scenario?

Reply:
That sounds good